src/pages/studyView/addChartButton/addChartByType/AddChartByType.tsx

```diff
@@ -114,7 +115,7 @@ export default class AddChartByType extends React.Component<
     }
 
     @computed get columns() {
-        const columns: Column<AddChartOption>[] = [
+        let columns: Column<AddChartOption>[] = [
```

Comment:
I don't think we are assigning something else to `columns`, let's keep it `const`.